### DISCUSSION TITLE
Residential Webshare.io proxies not working

### DISCUSSION THREAD

**Discussion by deleeuwpeter** [KevinBytesTheDust/CVAmp#199] [General]
First of all, thanks for creating this amazing tool! 
Unfortunately I can't get (non-private) residential proxies purchased via Webshare.io to work. In the log file I'm seeing either "Timeout 20000ms exceeded", "ERR_HTTP_RESPONSE_CODE_FAILURE at https://www.twitch.tv/login" or "ERR_TUNNEL_CONNECTION_FAILED".

Any idea what's going wrong? I'm following the[ guide you wrote](https://github.com/jlplenio/crude-twitch-viewer-bot/wiki/Webshare.io-Proxies-Guide) but using residential proxies instead of datacenter ones.
This to make the views appear more real and reduce chances of detection. But perhaps these residential proxy IP's have already been detected?

_FYI; the tool works fine with private datacenter proxies but they're quire expensive._

**Comment by jlplenio** [comment]
Hey Flapje. Thank you for the feedback. I have not tested residential proxies for this tool. They should work as normal, but are not recommended by me because they are far too expensive for the task at hand. As they are residential, they could be heavily limited in terms of traffic and concurrency. Please make sure to check, if you have enough traffic and concurrency here in Bandwidth and Errors:
![grafik](https://github.com/jlplenio/crude-twitch-viewer-bot/assets/16845311/0f95870d-50d6-4ba3-8fa1-1742503e156e)

**Reply by deleeuwpeter** [reply]
I suspect the limits of such proxies (not premium, not private) are causing the issues. Went back to datacenter proxies and everything is working fine again.

One more question; are the proxies in the proxies list randomly picked by the tool when spawning instances?

**Reply by dedalo88888** [reply]
Interesting final question :)
